recipes/py2app/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "py2app" %}
+{% set version = "0.21" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4516dfb01a0bf560ca4918e1e0e38eb4fcc76c9c2f527456e022310a80d52dc8
+
+build:
+  skip: true  # [not osx]
+  number: 0
+  entry_points:
+    - py2applet = py2app.script_py2applet:main
+  missing_dso_whitelist:
+    - '*/libgcc_s.1.dylib'
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - altgraph >=0.16
+    - macholib >=1.11
+    - modulegraph >=0.17
+    - python
+    - setuptools
```

Comment:
is setuptools really needed at run time by the user??

Reply:
One way to use this package is to build `pip` packages that are then put into a .app bundle for macOS, so `setuptools` is needed for that functionality (https://py2app.readthedocs.io/en/latest/dependencies.html). I'm using this package another way that can use `conda` environments in .app bundles.